tests/integration/targets/iosxr_vrf_interfaces/tests/common/_remove_config.yaml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Populate with simple overridden
+  cisco.iosxr.iosxr_vrf_interfaces:
```

Comment:
same here as per the last comment

Reply:
Not the same here, it's not required as the vrf name might be different in each case